remote-dependency-templates/README.md

```diff
@@ -3,9 +3,73 @@
   - Populate required `template-config` fields (defined in the README for each type)
 - List any dependency-specific service data under `dependency-data.<id>` in a service's `service.yml`
 
-## Terraform variables //TODO
+# Building dependency templates
+- Create a directory with a unique name. This will be the dependency's `type`
 
-## Notes
-The following are fields that exosphere passes to every dependency:
-- `app-name`: Name of the application as defined in `application.yml`
+## requirements.yml
+`requirements.yml` contains a single field: `required-fields`, which is a list of strings that define the fields that a user is required to populate
```

Comment:
Same here. The description doesn't need to mention the file name again. It can start with `Contains a single field`